examples/http-2.0-server-push/http_client.bal

```diff
@@ -75,7 +76,7 @@ public function main() {
     }
 
     // Fetch required promise responses.
-    foreach promise in promises {
+    foreach http:PushPromise promise in promises {
```

Comment:
```suggestion
    foreach var http:PushPromise promise in promises {
```

Reply:
This `promise` varibale is always `http:PushPromise`. So we can use the specific type. Isn't it?

Reply:
Yeah, I didn't notice the type has been added :)